connect.net

```diff
@@ -1,11 +1,13 @@
 ~0
-@1 2=0.00 3=0.00 5=0.00 6=0.00 11=0.00 50=0.00 509=0.00 900=0.00 1000=0.00 9999=0.00
+@1 2=0.00 3=0.00 5=0.00 6=0.00 11=0.00 50=0.00 100=0.00 509=0.00 900=0.00 1000=0.00 9999=0.00
 @2 1=0.00 9999=0.00
 @3 1=0.00
 @5 1=0.00
 @6 1=0.00
 @11 1=0.00
 @50 1=0.00
+@100 1=0.00
```

Comment:
If 101 should only talk to 100, please also add a connection to 101 here "101=0.00" is what you should add.

The format is awful, yes, but it's the easy legacy format